### DISCUSSION TITLE
Webdav failure -- catch 22

### DISCUSSION THREAD

**Discussion by LaurelMurphy** [iterate-ch/cyberduck#16455] [Q&A]
I am trying to upload to Big Commerce on windows.   The cyberduck I get error message "unexpected response 501, not implemented.  contact hosting service .  I have also tried to uninstall CyberDuck  and it says the installer is broken.  At this point I just want to wipe it all off my computer and start over.  Could someone help me with this?  Thank you.

**Comment by dkocher** [comment]
The error code `501` suggests a server error indicating you are not connecting to a WebDAV server. Please check your bookmark configuration for _Server_ with Big Commerce support or documentation.
